proposal/success.Rmd

```diff
@@ -18,20 +18,23 @@ The following can be used to measure success:
 
 - Can the package be installed from GitHub? (yes/no)
 - Can the package be installed from r-universe? (yes/no)
-- Test coverage (at least 90%)
-- Correctness tests for volatiltiy predictions including comparisons with measured values (± some tolerance) (yes/no)
-- Is the package passing R CMD check on linux, macOS, and windows using CI? (yes/no)
+- Is there sufficient test coverage? (at least 90%)
+- Are there correctness tests for volatility predictions including comparisons with measured values (± some tolerance) (yes/no)
+- Is the package passing `R CMD check` on Linux, macOS, and Windows using CI? (yes/no)
 - Has the package been tested by users other than the developers? (yes/no)
 - Does the package have a vignette that is easy to follow? (yes/no)
 - Is code archived on Zenodo and a DOI associated with the package citation? (yes/no)
-- The number of chemical representation types that can be used as input (molfile for success; inchikey, inchi, SMILES, and possibly more for reach goal)
+- Can chemical representation types be used as input? (molfile for success; InChI, SMILES, and possibly more for reach goal)
 
 ## Future work
 <!-- 
 How could this be extended / developed in the future by yourself and/or the community in general?
 -->
 
-**TODO: replace with text from future directions in manuscript in prep**
+- As a handful of functional groups relevant to volatility were not able to be measured using existing methods, the community who creates these methods could expand to these groups
+- The algorithm used in the package only calculates relative volatility under standard conditions; the algorithm could be added to or expanded to calculate volatility itself and for additional environmental conditions
+- As additional chemical compound databases are used as inputs to `volcalc`, these should be documented and either added to this package or created as standalone packages that link to `volcalc`
+- As this package is used increasingly by researchers, they will discover other edge cases or unusual compounds that the package can be improved to accomodate
```

Comment:
I don't think this is necessary here.  Could be part of the proposal somewhere else to mention that the Meredith lab and CCT will continue maintainance and development after this grant instead.
```suggestion
```